src/main/resources/public/Updated.html

```diff
@@ -318,89 +318,32 @@ <h1 id="computerWin" align="middle">You have lost!!!!</h1>
-                    <option value="9">9</option>
-                    <option value="10">10</option>
-                </select>-->
-                <br>
+                <br><br>
                 Orientation:
                 <select name="orientationSelec" id="orientationSelec">
                     <option value="horizontal">horizontal</option>
                     <option value="vertical">vertical</option>
                 </select>
-                <!-- <br>
-                 <button type="button" onclick="placeShip()">Place</button>-->
+                <br><br>
+                <h3>Computer Place options:</h3>
+                <select name="placeDiffSelec" id="placeDiffSelec">
+                    <option value="Easy">Easy</option>
+                    <option value="Hard">Hard</option>
+                </select>
             </div>
         </th>
 
 
         <th>
-            <!--<h3>Scan:</h3>-->
-                <input type="radio" name="fire" id="fireSelec" value="fireUpdated" checked>Fire<br>
-                <input type="radio" name="fire" value="scan">Scan<br>
-            <!---<div id="AircraftcarrierDiv">
-                Row:
-                <select name="rowFire" id="colFire">
-                    <option value="1">1</option>
-                    <option value="2">2</option>
-                    <option value="3">3</option>
-                    <option value="4">4</option>
-                    <option value="5">5</option>
-                    <option value="6">6</option>
-                    <option value="7">7</option>
-                    <option value="8">8</option>
-                    <option value="9">9</option>
-                    <option value="10">10</option>
-                </select>
-                <br>
-                Column:
-                <select name="colFire" id="rowFire">
-                    <option value="1">1</option>
-                    <option value="2">2</option>
-                    <option value="3">3</option>
-                    <option value="4">4</option>
-                    <option value="5">5</option>
-                    <option value="6">6</option>
-                    <option value="7">7</option>
-                    <option value="8">8</option>
-                    <option value="9">9</option>
-                    <option value="10">10</option>
-                </select>
-                <br>
-                <button type="button" onclick="fire()">Scan</button>
-            </div>-->
+            <h3>Player fire options:</h3>
+            <input type="radio" name="fire" id="fireSelec" value="fireUpdated" checked>Fire<br>
+            <input type="radio" name="fire" value="scan">Scan<br>
+            <br><br>
+            <h3>Computer Fire options:</h3>
+            <select name="fireDiffSelec" id="fireDiffSelec">
+                <option value="Easy">Easy</option>
+                <option value="Hard">Hard</option>
+            </select>
```

Comment:
The value should be 1: for Easy and 0: for hard.
This will never work on the backend.